pyfixest/did/lpdid.py

```diff
@@ -9,6 +9,37 @@
 
 
 class LPDID(DID):
+    """
+    A class used to represent the Local Polynomial Differences-in-Differences model.
```

Comment:
Here it should say the "local projections"  estimator. Took me a while, I'd always say "linear projections" instead ... 😄